### PR TITLE
Adding GTS Formatting

### PR DESCRIPTION
**Pull request by schmidt-sebastian**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,8 +39,6 @@
     "cover": "nyc --reporter=lcov mocha --require intelli-espower-loader build/test/*.js && nyc report",
     "docs": "jsdoc -c .jsdoc.js",
     "generate-scaffolding": "repo-tools generate all",
-    "lint": "eslint src/ system-test/ test/ conformance/",
```
Comment (JustinBeckwith) [inline package.json]: We kept these around in other repos because of samples.  Does this repo keep samples in it?

Comment (schmidt-sebastian) [reply package.json]: It does, but they have their own package.json:

https://github.com/googleapis/nodejs-firestore/blob/master/samples/package.json

Should I add it back either way?

**Comment by schmidt-sebastian** [comment]
BTW, the Node 4 tests are now failing because of: 

```
/home/node/project/node_modules/gts/node_modules/meow/index.js:66
	const {pkg} = options;
```

Should we disable the Node4 runner (and the lint runner?) and abandon Node 4? It would allow me to add the spread operator back :-)

**Comment by JustinBeckwith** [comment]
Yes!

**Comment by schmidt-sebastian** [comment]
I removed Node4/Lint from CI, but it looks like I can't remove the checks from GitHub. @JustinBeckwith can you do this?

**Comment by JustinBeckwith** [comment]
Apologies for not saying this out loud, but we should really disable nodejs 4 and 9 in a different pull request (just to keep the commits separate)

**Comment by schmidt-sebastian** [comment]
I moved the commits that disabled Node 4 to #241

**Comment by schmidt-sebastian** [comment]
Merged package-lock.json. This still seems to be blocked on the `lint` step.

**Pull request merged**